frontend/src/store/transaction.js

```diff
@@ -49,30 +50,84 @@ export default ({
       context.commit('setUser', response.data)
     },
     getRegisteredNodes (context, params) {
-      let page = params.page || context.state.nodePage
+      let page = context.state.nodePage
```

Comment:
this will always be 1 here no ? since we always get all the nodes ?

Reply:
No, it will increment if there are more pages https://github.com/threefoldtech/tfexplorer/blob/b2b43649918dd387125ee886485db85bbf08f05e/frontend/src/store/transaction.js#L157